### PR TITLE
Sklearn fixes

### PR DESCRIPTION
**Pull request by animeshsingh**
Changing SKLearn to take Lists as input/output as well as adding tests.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/91)

### CODE REVIEW

File: python/sklearnserver/sklearnserver/model.py [hunk 1]
```diff
@@ -2,21 +2,27 @@
...
+        kfserving.Storage.download(self.model_dir),JOBLIB_FILE)
         self._joblib = joblib.load(model_file)
         self.ready = True
 
-    def predict(self, inputs):
+    def predict(self, body: List) -> List:
         try:
-            inputs = np.array(inputs)
+            inputs = np.array(body)
+            print(inputs)
```
Comment (ellis-bigelow) [inline python/sklearnserver/sklearnserver/model.py]: toss this print.

File: python/sklearnserver/README.md [hunk 2]
```diff
@@ -36,3 +36,71 @@ __main__.py: error: the following arguments are required: --model_dir
...
+
+sklearnserver/test_model.py .                                                                                             [100%]
+
+=================================================== 1 passed in 0.43 seconds ====================================================
+```
+
+To run static type checks:
+
+```bash
+mypy --ignore-missing-imports xgbserver
```
Comment (ellis-bigelow) [inline python/sklearnserver/README.md]: Can we put this in the makefile?. Also xgbserver -> sklearnserver

File: python/sklearnserver/sklearnserver/model.py (hunk 1 above)
Comment (animeshsingh) [reply python/sklearnserver/sklearnserver/model.py]: Done

File: python/sklearnserver/README.md (hunk 2 above)
Comment (animeshsingh) [reply python/sklearnserver/README.md]: xgbserver -> sklearnserver - Done.

The test is in makefile. I can remove the description here if you think so

Comment (ellis-bigelow) [reply python/sklearnserver/README.md]: Oh I was suggesting we just run it as part of the test flow. Kind of like a "vet"

Comment (animeshsingh) [reply python/sklearnserver/README.md]: Done

**Comment by ellis-bigelow** [comment]
Great work
/lgtm

**Pull request merged**